File: go/3_math/4_prima_kex.go
package main

import (
	"fmt"
	"math"
)

func Prima(N int) bool {
	if N < 2 {
		return false
	}
	for i := 2; i <= int(math.Sqrt(float64(N))); i++ {
		if N%i == 0 {
			return false
		}
	}
	return true
}

func GetPrime(X int) int {
	primeNumber := 2
	counter := 0
	for counter < X {
		if Prima(primeNumber) == true {
			counter++
		}
		if counter != X {
			primeNumber++
		}
	}
	return primeNumber
}

func main() {
	fmt.Println(GetPrime(1))  // 2
	fmt.Println(GetPrime(5))  // 11
	fmt.Println(GetPrime(8))  // 19
	fmt.Println(GetPrime(9))  // 23
	fmt.Println(GetPrime(10)) // 29
}

File: python/3-math/4-prima-ke-x.py
def primeNumber(N):
  if N <= 1:
    return False
  for i in range(2, N):
    if N % i == 0:
      return False
  return True

def getPrime(numberX):
  primeX = 1
  counter = 0
  while True:
    if primeNumber(primeX):
      counter += 1
    if counter == numberX:
      break
    primeX += 1
  return primeX

print(getPrime(1)) # 2
print(getPrime(5)) # 11
print(getPrime(8)) # 19
print(getPrime(9)) # 23
print(getPrime(10)) # 29
print(getPrime(168)) # 997
print(getPrime(1000)) # 7919